sdk/deviceprovisioningservices/azure-resourcemanager-deviceprovisioningservices/pom.xml

```diff
@@ -63,6 +63,12 @@
         <version>2.4.0</version> <!-- {x-version-update;com.azure.resourcemanager:azure-resourcemanager-resources;dependency} -->
         <scope>test</scope>
       </dependency>
+      <dependency>
+        <groupId>com.azure.resourcemanager</groupId>
+        <artifactId>azure-resourcemanager-test</artifactId>
+        <version>2.0.0-beta.1</version> <!-- {x-version-update;com.azure.resourcemanager:azure-resourcemanager-test;current} -->
```

Comment:
You should not include this.

If you really really need this, you need to put it in a profile, as this `azure-resourcemanager-test` is not released to maven.

```
    <profile>
      <id>azure-mgmt-sdk-test-jar</id>
      <activation>
        <property>
          <name>!maven.test.skip</name>
        </property>
      </activation>
      <dependencies>
        <dependency>
          <groupId>com.azure.resourcemanager</groupId>
          <artifactId>azure-resourcemanager-test</artifactId>
          <version>2.0.0-beta.1</version> <!-- {x-version-update;com.azure.resourcemanager:azure-resourcemanager-test;current} -->
          <scope>test</scope>
        </dependency>
      </dependencies>
    </profile>
```